### PR TITLE
Testing slack integration

### PR DESCRIPTION
**Pull request by stilesj-uchicago**
### Improvements
Testing the slack integration for the 3rd time.

### CODE REVIEW

File: README.slack.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Github slack integration: https://github.com/integrations/slack
```
Comment (kulgan) [inline README.slack.md:1]: Nice

**Review by kulgan — COMMENTED** [review]
Just testing again

File: README.slack.md (hunk 1 above)
Comment (stilesj-uchicago) [reply README.slack.md:1]: I appreciate the time.

File: README.slack.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+1. Add the App to Slack
+1. Login `/github signin`
```
Comment (stilesj-uchicago) [inline README.slack.md:3]: I am starting a review!

**Pull request closed without merging**